nemoguardrails/library/sensitive_data_detection/actions.py

```
@@ -40,15 +44,15 @@ def _get_analyzer(score_threshold: float = 0.4):
     if not 0.0 <= score_threshold <= 1.0:
         raise ValueError("score_threshold must be a float between 0 and 1 (inclusive).")
     try:
-        from presidio_analyzer import AnalyzerEngine
+        from presidio_analyzer import AnalyzerEngine  # type: ignore[reportMissingImports]
 
     except ImportError:
         raise ImportError(
             "Could not import presidio, please install it with `pip install presidio-analyzer presidio-anonymizer`."
         )
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)